radix-transactions/src/manifest/static_resource_movements/typed_invocation.rs

```diff
@@ -313,6 +314,19 @@ macro_rules! define_manifest_typed_invocation {
                 }
             }
             pub use uniform_match_on_manifest_typed_invocation;
+
+            /// This is a function that's there for testing purposes only.
+            pub fn typed_native_invocation_function_table() -> HashMap<&'static str, HashMap<&'static str, HashSet<&'static str>>> {
```

Comment:
Putting this behind `#[cfg(test)]` or `#[doc(hidden)]` is typical if it's just exposed for testing purposes.

Reply:
This doesn't work since the `radix-transactions` is not compiled with the test flag in tests: https://users.rust-lang.org/t/how-to-import-a-function-from-tests-in-another-file/88981/11?u=0xomara

Reply:
Did you try `#[doc(hidden)]`?